packages/backend/.eslintrc.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   },
   plugins: ['@typescript-eslint/eslint-plugin'],
   extends: [
+    'eslint:recommended',
```

Comment:
Is this change necessary? The other two seem to be sufficient guidelines seem to be sufficient.

Reply:
This change is made based on another pr made by @j-chad . Commenting this out seems not creating issues.